src/layout/layoutinstance.ts

```diff
@@ -884,10 +884,13 @@ export class LayoutInstance {
                 let targetNodeId = tuple[1];
 
                 directions.forEach((direction) => {
-                    // Only add alignment edge if enabled
+                    // Only add alignment edge if enabled AND edge doesn't already exist in the graph
                     if (direction.startsWith("directly") && this.addAlignmentEdges) {
                         const alignmentEdgeLabel = `_alignment_${sourceNodeId}_${targetNodeId}_`;
-                        g.setEdge(sourceNodeId, targetNodeId, alignmentEdgeLabel, alignmentEdgeLabel);
+                        const graphHasAlignmentEdge = g.hasEdge(sourceNodeId, targetNodeId, alignmentEdgeLabel);
```

Comment:
I *think* you can also not add an alignment edge if the symmetric edge is in the graph as well. That is, if a -> b or b->a is in the graph, dont add it.

Reply:
Also, I don't think you need to add the alignment edge label. *Any* edge's presence should mean we dont need another edge.